content/docs/reference-react-component.md

```diff
@@ -37,9 +37,9 @@ class Welcome extends React.Component {
 >
 >React'te, ES6 sınıfı olarak bileşen tanımlamak zorunda değilsiniz. Eğer ES6 kullanmak istemiyorsanız, npm paketi olarak yer alan [create-react-class](/docs/creact-without-es6.html) modülü veya benzer bir özel soyutlama yöntemi kullanabilirsiniz. 
 
-### The Component Lifecycle {#the-component-lifecycle}
+### Bileşenin Yaşam Döngüsü {#the-component-lifecycle}
 
-Each component has several "lifecycle methods" that you can override to run code at particular times in the process. **You can use [this lifecycle diagram](http://projects.wojtekmaj.pl/react-lifecycle-methods-diagram/) as a cheat sheet.** In the list below, commonly used lifecycle methods are marked as **bold**. The rest of them exist for relatively rare use cases.
+Her bileşen, belirli anlarda çalıştırabileceğiniz birkaç "yaşam döngüsü metodu" (lifecycle methods) sunar. **Bu metodları hatırlamak için, [yaşam döngüsü diyagramını](http://projects.wojtekmaj.pl/react-lifecycle-methods-diagram/) kullanabilirsiniz.** Aşağıdaki listede, yaygın olarak kullanılan yaşam döngüsü metodları **kalın** harfler ile belirtilmiştir. Geri kalan metotlar, daha nadir kullanımlar için uygundur.
```

Comment:
yaşam döngüsü diyagramı - biyolojik yaşama işaret etmiyor. "React kod söz dizimi(syntax) metodu " işaret edilerek (kodalama jargonu olmalı - native çevirme kafa karıştıracaktır!)  çevirme yapılırsa daha açıklayıcı olur. 
"React Kodyazımı" veya "Kodyazım Akışı" gibi...

Reply:
Merhaba,
Lifecycle terimi, sadece biyolojik yaşamlar için geçerli değildir. Örnek kullanımlar:
* http://blog.microsoft.com.tr/?p=3715
* http://www.informatikplm.com/blog/integrity-lifecycle-manager/gereksinim-ve-dogrulama-uygulama-yasam-dongusu-yonetimi.
* http://www.ugurkizmaz.com/YazilimMakale-1825-Yazilim-Yasam-Dongusu-Nedir.aspx